<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinLoad</class>
 <widget class="QDialog" name="WinLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #3a3633;
}</string>
  </property>
  <widget class="QLabel" name="lbl_readers">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Читатели:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_readersPhoto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Фото читателей:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_booksPhoto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Фото книг:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_books">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Книги:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_booksTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 9pt &quot;Tahoma&quot;;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Название таблицы книг:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_readers">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_books">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_readersPhoto">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_booksPhoto">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_booksTable">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
border-width:1px;
border-color: #3a3633;
border-style: solid;
font: 10pt &quot;Tahoma&quot;;
color: white;
selection-color: white;
selection-background-color: #d81b1b;
background-color: #524E4B;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_readers">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_readersPhoto">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_books">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_booksPhoto">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #d81b1b;
border-width:0px;
border-color: #3a3633;
border-style: solid;
padding: 5px;
padding-left: 5px;
padding-right: 5px;
font: 10pt &quot;Tahoma&quot;;
color: white;
}
QPushButton:pressed {
background-color:rgba(255,255,255,.2);
}
QPushButton:hover { background: rgb(255,37,37); }
QPushButton:disabled { background:#3a3633; }</string>
   </property>
   <property name="text">
    <string>Загрузить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
